edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.CloudProxy.Test/CloudProxyTest.cs

```diff
@@ -125,12 +155,12 @@ public async Task CanListenForDesiredPropertyUpdates()
         {
             var update = new TaskCompletionSource<IMessage>();
             var edgeHub = new Mock<IEdgeHub>();
-            string deviceConnectionStringKey = "device2ConnStrKey";
+            string deviceConnectionStringKey = Device2ConnStrKey;
```

Comment:
nit: remove deviceConnectionStringKey

Reply:
I will remove here and where appropriate